charts/kof-child/templates/child-multi-cluster-service.yaml

```
@@ -66,3 +66,9 @@ spec:
         kind: ConfigMap
         name: kof-cluster-config-{{`{{ .Cluster.metadata.name }}`}}
         namespace: "{{`{{ .Cluster.metadata.namespace }}`}}"
+    - resource:
+        apiVersion: cluster.x-k8s.io/v1beta1
+        kind: Cluster
+        name: "{{`{{ .Cluster.metadata.name }}`}}"
+        namespace: "{{`{{ .Cluster.metadata.namespace }}`}}"
+      identifier: MyCluster
```

Please add `optional: true`, otherwise it doesn't work on adopted clusters.

```suggestion
    - resource:
        apiVersion: cluster.x-k8s.io/v1beta1
        kind: Cluster
        name: "{{`{{ .Cluster.metadata.name }}`}}"
        namespace: "{{`{{ .Cluster.metadata.namespace }}`}}"
      identifier: ChildCluster
      optional: true
```